emails/templates/missed-vote.js

```diff
@@ -9,6 +9,7 @@ const { accountData } = require('../mock-utils')
 
 module.exports = function() {
   return {
+    subject: 'Your vote wasn’t casted on time',
```

Comment:
```suggestion
    subject: 'Your vote wasn’t cast on time',
```

Escaping the wrath of John one `casted` at a time 💨